src/modules/tests/tests.effects.ts

```diff
@@ -206,6 +209,15 @@ export class TestsEffects {
         startTestAction.category === TestCategory.DE) {
         arrayOfActions.push(new InitializeVehicleChecksCatD(startTestAction.category));
       }
+      if (
+        startTestAction.category === TestCategory.F ||
+        startTestAction.category === TestCategory.G ||
+        startTestAction.category === TestCategory.H ||
+        startTestAction.category === TestCategory.K) {
+        arrayOfActions.push(new GearboxCategoryChanged('Manual'));
+        arrayOfActions.push(new RouteNumberChanged(88));
+        arrayOfActions.push(new IndependentDrivingTypeChanged('N/A'));
```

Comment:
Spacing seems off, might cause a linting issue

Reply:
Doesnt seem to be an issue, it was in that format before i edited it